conf/test_epicore.config

```diff
@@ -0,0 +1,35 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running epicore
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/mhcquant -profile test_epicore,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+// Limit resources so that this can run on GitHub Actions
+process {
+    resourceLimits = [
+        cpus: 2,
+        memory: '6.GB',
+        time: '2.h'
+    ]
+}
+
+params {
+    config_profile_name        = 'Test epicore profile'
+    config_profile_description = 'Minimal test dataset to check pipeline function with epicore'
+
+    // Input data
+    input = params.pipelines_testdata_base_path + 'mhcquant/testdata/HepG2_sample_sheet.tsv'
+    fasta = params.pipelines_testdata_base_path + 'mhcquant/testdata/UP000005640_9606.fasta'
+
+    // Pipeline settings
+    epicore                                  = true
+    epicore_min_overlap                      = '11'
```

Comment:
Epicore settings are already set by default in nextflow.config. No need to specify them here again.